### ISSUE TITLE
Wrong port when loading dumps

### ISSUE THREAD

**Issue by tessi** [makandra/geordi#30]
After updating geordi to 1.6.2 (especially after [this commit](https://github.com/makandra/geordi/commit/d0abf159c14fc82311378d077fc3cf5db3f984ff)) I get an entertaining error when loading dumps:

```
geordi dump -l tmp/production.dump

# Sourcing dump into the secret db
> Source file: tmp/production.dump
pg_restore: [archiver (db)] connection to database "secret" failed: invalid port number: "඀"

x An error occured loading production.dump
```

putting a debugger into geordi, just before it calls `pg_restore` and asking for the `config` variable leads to:

```
{
  "adapter"=>"postgresql",
  "username"=>"secret",
  "password"=>"secret",
  "encoding"=>"unicode",
  "pool"=>5,
  "port"=>3456,
  "host"=>"localhost",
  "database"=>"secret"
}
```

So, the port is read correctly from my `database.yml`.

The bug is entertaining, because I can change my port to be `9731` leading to the following error:

```
pg_restore: [archiver (db)] connection to database "secret" failed:
invalid port number: "☃"
```

It works if I specify the port as a string (in `database.yml`) **You can use that as a workaround**.

I *guess* geordi somehow takes the port-integer and converts it to a character before passing it to the geordi executable.

**Comment by kratob** [comment]
At least you got some enjoyment out of it. ;-)

Should be fixed in 1.6.3.

**Comment by tessi** [comment]
⛄️ 

thanks for fixing it!
